File: stn-homa/src/app/domain/costs/cost.ts
export class Cost {
    constructor(
        public id: number,
        public name: string,
        public workshop: string,
        public street: string,
        public number: number,
        public zipCode: number,
        public city: string,
        public date: Date,
        public mileage: number,
        public price: number,
        public comment: string
    ) { }
}

File: Stn.Homa/Stn.Homa.Fleet.Api/Entities/Cost.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Stn.Homa.Fleet.Api.Entities
{
    public class Cost
    {
        public int Id { get; set; }

        public string CarName { get; set; }

        public string Name { get; set; }

        [Column("ServiceName")]
        public string WorkshopName { get; set; }

        public DateTime Date { get; set; }

        public double? Mileage { get; set; }

        public decimal? Price { get; set; }

        [Column("Comments")]
        public string Comment { get; set; }
    }
}
